backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/tourplanning/api/impl/DeliveryDayDAO.java

```diff
@@ -74,7 +74,7 @@
 				.addOnlyActiveRecordsFilter()
 				//
 				// for given BP & Location
-				.addInArrayFilter(I_M_DeliveryDay.COLUMNNAME_C_BPartner_ID , null, params.getBPartnerLocationId().getBpartnerId())
+				.addInArrayFilter(I_M_DeliveryDay.COLUMNNAME_C_BPartner_ID, null, params.getBPartnerLocationId().getBpartnerId())
```

Comment:
what if `params.getBPartnerLocationId()` is null?

Reply:
true. Fixed